### PR TITLE
Add support for ZigBee Home Automation

### PR DESCRIPTION
**Pull request by rcloran**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#6263

### CODE REVIEW

File: source/_components/zha.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- [Sensor](../sensor.zha) (e.g. temperature sensors)
+- [Light](../light.zha)
+- [Switch](../switch.zha)
+
+Known working ZigBee radios:
+
+- Nortek/GoControl Z-Wave & Zigbee USB Adaptor - Model HUSBZB-1
+
+To configure the component, a `zha` section must be present in the `configuration.yaml`,
+and the path to the serial device for the radio and path to the database which
```
Comment (Landrash) [inline source/_components/zha.markdown]: , & and.

**Comment by balloob** [comment]
@rcloran bump, got time to address these changes so we can merge the docs for the next version?

**Comment by Landrash** [comment]
@rcloran Wan't me to implement the changes I've suggested?

**Comment by rcloran** [comment]
Argh, I keep doing other things by mistake. There are a few other changes I'd like to make, but if you have time, please go ahead and do this. Otherwise I'll get to it within 24 hours.

**Comment by Landrash** [comment]
@rcloran No worries. I'll do it later today :)

File: source/_components/zha.markdown [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# Example configuration.yaml entry
+zha:
+  usb_path: /dev/ttyUSB2
+  database_path: zigbee.db
+```
+
+Configuration variables:
+
+ - **usb_path** (*Required*): Path to the serial device for the radio.
+ - **database_path** (*Required*): Path to the database which will keep persisten newtork data.
```
Comment (Landrash) [inline source/_components/zha.markdown]: @rcloran Will you look over this section. Added it from the information in the post but unsure about how correct it is.

**Pull request merged**